pages/api/create_order/index.js

```diff
@@ -0,0 +1,66 @@
+import axios from "axios";
+import CryptoJS from "crypto-js";
+import moment from "moment";
+import { configZLP } from "../config";
+
+export default async function handler(req, res) {
+  if (req.method !== 'POST') {
+    res.setHeader('Allow', 'POST');
+    res.status(405).end('Method Not Allowed');
+    return;
+  }
+
+  try {
+    const paymentType = req.body.paymentType
+    let embed_data = { redirecturl: "http://localhost:3000/result" };
+    const items = [{}]; // todo: collect items from Cart page
+    const transID = Math.floor(Math.random() * 1000000);
+    const appTransID = `${moment().format('YYMMDD')}_${transID}`;
+
+    // Handle payment type 
+    let bankCode = ""
+
+    if (paymentType === "ZALOPAY") {
+      bankCode = "zalopayapp"
+    }
+    if (paymentType === "CC") {
+      bankCode = "CC"
+    }
+    if (paymentType === "ATM") {
+      embed_data.bankgroup = "ATM"
+    }
+
+    if (paymentType !== "ALL") {
+      bankCode = paymentType
+    }
+
+    const order = {
```

Comment:
we still agree to hard code those order information?

Reply:
I will adjust the BE API to handle the real amount number